src/multi_physics/QED/src/math/quadrature.hpp

```diff
@@ -9,11 +9,11 @@
 // instabilities of the tanh_sinh integration method
 #define BOOST_ENABLE_ASSERT_HANDLER
 #include <boost/assert.hpp>
-#undef BOOST_ENABLE_ASSERT_HANDLER
 #include <boost/math/quadrature/trapezoidal.hpp>
 #include <boost/math/quadrature/tanh_sinh.hpp>
 #include <boost/math/quadrature/exp_sinh.hpp>
 #include <boost/math/quadrature/gauss_kronrod.hpp>
+#undef BOOST_ENABLE_ASSERT_HANDLER
```

Comment:
slightly more robust:
```C++
#ifdef BOOST_ENABLE_ASSERT_HANDLER
#   undef BOOST_ENABLE_ASSERT_HANDLER
#endif
```
(in case this changes upstream in the future)

Reply:
oh wait, you define it above.